resources/debugger/components/elements/element-tree-item.jsx

```diff
@@ -116,12 +119,32 @@ export default class ElementTreeItem extends Component {
     )}</Info>)
   }
 
+  propagateClick(e) {
+    var target = ReactDOM.findDOMNode(e.target);
+    if (target.tagName == 'LI') {
+      var toggleButton = target.getElementsByTagName('button')[0];
+      if (toggleButton) {
+        toggleButton.click()
+      }
+    }
+    else if (target.tagName == 'SPAN') {
+      target = target.closest('li');
+      var toggleButton = target.getElementsByTagName('button')[0];
+      if (toggleButton) {
+        toggleButton.click()
+      }
+    }
+  }
```

Comment:
I'm not sure about that, it seems rather fragile.

How about changing the line 163 to ```
<WrapElement>
  <span onClick={() => this.setState({ expanded: !this.state.expanded })}>&lt;{this.renderElementName()} /&gt;</span> {this.renderQuickLook()}
</WrapElement>
```
and removing this click handler?

Reply:
I didn't figure out this one yet, I think it's something related to which element captures the click event depending on where you click (sometimes is the li, sometimes the span, …).

Also this might be related to the topic of auto-dismissing the current QuickLook sidebar when a new one is called (as I imagine we want those click events to either expand the tree-view or open the QuickLook if the element has no children).

In any case I'll take out the hacky propagateClick() by now until we figure it out 👍